TAInformation/Models/admin.py

```diff
@@ -1,6 +1,32 @@
 from TAInformation.Models.base_user import BaseUser
+from django.contrib import admin
 from TAInformation.Models.account_type import AccountType
-from TAInformation.models import Course, User, LabCourseJunction, Lab
+from TAInformation.models import Course, User, LabCourseJunction, Lab, taAssignment, courseName
+
+admin.site.register(taAssignment)
+
+
+def display_taAssignment():
+
+    all_assignment = Course.objects.all()
+    assignment_content = []
+    for course in all_assignment:
+        all_assignment_ids = LabCourseJunction.objects.filter(course_id=all_assignment_ids)
```

Comment:
What would be the value of 'all_assignment_ids' when using it as filter criteria?  If you're trying to create a list of all assignments ids in the table, you should do something like this:

LabCourseJunction.objects.values_list('lab_id', flat=True).distinct()

More details can be read here: https://stackoverflow.com/questions/37205793/django-values-list-vs-values